great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -403,86 +401,6 @@ def update_project_config(
         self.config.update(project_config)
         return self.config
 
-    @public_api
-    @deprecated_method_or_class(
-        version="0.15.48", message="Part of the deprecated DataContext CRUD API"
-    )
-    @usage_statistics_enabled_method(
-        event_name=UsageStatsEvents.DATA_CONTEXT_SAVE_EXPECTATION_SUITE,
-        args_payload_fn=save_expectation_suite_usage_statistics,
-    )
-    def save_expectation_suite(
```

Comment:
We still use this in our cloud quickstart.
https://docs.greatexpectations.io/docs/gx_cloud/tutorials/getting_started/getting_started_with_gx_cloud/#32-add-expectation